exporter.go

```diff
@@ -115,9 +115,18 @@ func (e *Exporter) Export(opts ExportOptions) (files.Report, error) {
 	meta.RunImage.Reference = opts.RunImageRef
 
 	if e.PlatformAPI.AtLeast("0.12") {
+		// if it's new API version but only using Stack still, then normalize the model by copying stack into the RunImage
+		if opts.RunImageForExport.Image == "" && len(opts.RunImageForExport.Mirrors) == 0 {
+			opts.RunImageForExport.Image = opts.Stack.RunImage.Image
+			opts.RunImageForExport.Mirrors = opts.Stack.RunImage.Mirrors
+		}
```

Comment:
Technically this backwards-compat layer is happening at builder creation, but by the time we get to the lifecycle I think it makes sense to just use the information from run.toml if we're on Platform API 0.12